twisted/conch/client/default.py

```diff
@@ -246,18 +255,46 @@ def getPrivateKey(self):
 
     def getGenericAnswers(self, name, instruction, prompts):
         responses = []
-        try:
-            oldout, oldin = sys.stdout, sys.stdin
-            sys.stdin = sys.stdout = open('/dev/tty','r+')
+        with self._replaceStdoutStdin():
             if name:
-                print(name)
+                print(name.decode("UTF8"))
```

Comment:
In this interests of consistency, this should be `"utf-8"` (117 occurrences in the codebase) not `"UTF8"` (24 occurrences, only one of which is in an `.encode` or `.decode` call.)  I also dimly recall that choice was intentional because there are some common misconfigurations of Python which will cause "UTF8" to break but "utf-8" works everywhere.